tests/microos/toolbox.pm

```diff
@@ -129,8 +137,12 @@ sub run {
     if (!validate_script_output 'toolbox list', sub { m/devel/ }, timeout => 180, proceed_on_failure => 1) {
         record_info('ISSUE', 'https://github.com/kubic-project/microos-toolbox/issues/23');
     }
-    assert_script_run 'toolbox run -c devel -- zypper lr';
-    assert_script_run 'toolbox run -c devel -- zypper -n in python3', timeout => 180 unless is_sle_micro;
+    my $output = script_output('toolbox run -c devel -- zypper lr', timeout => 180, , proceed_on_failure => 1);
+    if ($output =~ m/No repositories defined/ && is_sle_micro('=6.0')) {
```

Comment:
It would be nice to encapsulate this condition into a subroutine as it is re-used

Reply:
done